### PR TITLE
Add documentation for new connectors

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+The following lists the connections you can use to interact with generic protocol-based APIs:
+  
+ - **HTTP**: Communicates with external endpoints over HTTP.   
+ - **SMTP**: Performs email-related operations over SMTP. 
+ - **POP3**: Retrieves emails from a mail server over POP3. 
+ - **IMAP**: Retrieves emails from a mail server over a TCP/IP connection.
+
+
```
Comment (shaniR) [inline en/docs/reusable-components/choreo-connectors-list.md:8]: I guess we can reduce the space here...

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ - **NetSuite**: Integrate with Netsuite to perform operations such as creating a record instance, creating a sublist, etc.
+ - **Azure EventHub**: Integrate with Azure to manage consumer groups, event hubs, events, publishers, and partitions. You can perform many operations for the mentioned items including getting, listing, creating, deleting, etc.
+ - **Azure CosmosDB**: Integrate with Azure CosmosDB to manage documents and stored procedures. You can perform operations for documents and stored procedures such as getting, creating, querying, replacing, deleting, etc.
+ - **MongoDB**: Integrate with MongoDB to manage databases, collections, and documents. You can perform operations such as getting databases, collections, and documents, inserting, updating, and deleting documents, closing the client, listing indices, etc. 
+ - **Redis**: Integrate with Redis to manage different data structures. You can perform a range of operations depending on the data structure.
+ - **AWS S3**: Integrate with AWS S3 to manage cloud storages. You can perform operations such as creating, listing and deleting buckets as well as objects.
```
Comment (shaniR) [inline en/docs/reusable-components/choreo-connectors-list.md:25]: Missing the oxford comma.

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ - **AWS SQS**: Integrate with AWS SQS to manage messages and queues. You can perform operations such as creating queues, receiving messages, sending messages, deleting messages, and deleting queues.
+ - **COVID 19 API**: Integrate with the COVID 19 API to retrieve statistics relating to COVID-19. The GET operations you can perform include retrieving the status relating to the spread of COVID-19 and relating to the vaccine coverage per province, country, and continent. You can also retrieve global data.
```
Comment (shaniR) [inline en/docs/reusable-components/choreo-connectors-list.md:27]: should it be COVID 19 or COVID-19?

Comment (shaniR) [reply en/docs/reusable-components/choreo-connectors-list.md:27]: related to?

Comment (shaniR) [reply en/docs/reusable-components/choreo-connectors-list.md:27]: Shall we rephrase this to say " You can perform operations such as retrieving the statistics regarding the spread of the virus in USA and globally. You can also retrieve statistics relating to the vaccine coverage per province, country, and continent."

File: en/mkdocs.yml [hunk 4]
```diff
@@ -97,6 +98,9 @@ markdown_extensions:
     - pymdownx.snippets:
           base_path: docs
     - attr_list
+    - markdown_include.include:
+          base_path: docs
+
```
Comment (shaniR) [inline en/mkdocs.yml:103]: Let's remove this extra line.

File: en/mkdocs.yml [hunk 5]
```diff
@@ -124,3 +128,5 @@ extra:
     site_version: 1.0.0
     #base_path: http://localhost:8000
     base_path: http://localhost:8000
+
```
Comment (shaniR) [inline en/mkdocs.yml:131]: Let's remove these extra lines.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/references/choreo-connectors.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Choreo Connectors
+
+Connectors in Choreo, exposed as API calls, allow you to connect to external services through your Choreo service. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
```
Comment (Nashaath) [inline en/docs/references/choreo-connectors.md:3]: Since this content is going to be common, I suppose it can cause confusion when we say "through your Choreo service" when this content is displayed in integration concepts.

Comment (anupama-pathirage) [reply en/docs/references/choreo-connectors.md:3]: This will become a repetitive content as we have a similar thing under API Call topic in each md file

File: en/docs/reusable-components/choreo-connectors-list.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/reusable-components/choreo-connectors-list.md:27]: The name of the API in "COVID 19" without the dash, but in general, it is written with the dash.

Comment (Nashaath) [reply en/docs/reusable-components/choreo-connectors-list.md:26]: Created an issue a couple of days back to get Covid 19 changed to COVID-19 [1]. This task is in progress at the moment.
[1] https://github.com/wso2-enterprise/choreo/issues/5680

**Comment by anupama-pathirage** [comment]
@RukshiW This PR contains root cause analysis doc related changes as well. Please send different PRs for different things.

**Comment by RukshiW** [comment]
https://github.com/wso2/docs-choreo-dev/pull/193 makes this obsolete
